clients/tests/retrieval_client_test.go

```diff
@@ -264,9 +265,10 @@ func TestValidBlobHeader(t *testing.T) {
 	indexer.On("GetObject", mock.Anything, 1).Return(operatorSocket, nil).Once()
 
 	data, err := retrievalClient.RetrieveBlob(context.Background(), batchHeaderHash, 0, 0, batchRoot, 0)
-	assert.NoError(t, err)
+	require.Nil(t, err)
```

Comment:
Why this change?

Reply:
it returns error immediately, but I can change it back